### PR TITLE
Scripts for build and run QEMU

### PR DESCRIPTION
**Pull request by pgsamila**
Script `build_qemu.sh` will build and install QEMU. Scripts `build_kernel4.x.0.sh` will build kernel for the beta image. Script `download_axiom_beta_image.sh` will download the image files. Script `turn_on_qemu.sh` will turn on the QEMU.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,55 @@
...
+Follow the following steps to build and run the QEMU.
+
+# Step 1: Build and Installing QEMU 
+First you need to build and install the QEMU. For that, go open the terminal on axiom-beta-qemu repo folder and run the following commands.
+
+`chmod +x build_qemu.sh`
+
+`./build_qemu.sh`
+
+Note: This will download QEMU form Xilinx QEMU repo. Therefore you need to have internet conneciton while running this script. Also provide system password to install QEMU.
```
Comment (sebix) [inline README.md:14]: > QEMU form Xilinx

from *the* Xilinx...
s/form/from

> Also provide system password

*the* system password
> conneciton

connection

File: README.md [hunk 2]
```diff
@@ -1,2 +1,55 @@
...
+
+`chmod +x build_kernel4.6.0.sh`
+
+`./build_kernel4.6.0.sh`
+ 
+Note: This will download Xilinx linux kernel newest verion. Therefore you need to have a good internet connection while running this script.
+
+# Step 2-[B]: Build Xilinx Kernel 4.9.0
+Note: You need to follow either Step 2-[A] or 2-[B].
+This step will build Xilinx Kernel 4.9.0. How ever at the moment, this kernel verion will not run on QEMU properly. Recomend to use Kernel 4.6.0.
```
Comment (sebix) [inline README.md:29]: > Recomend to use Kernel 4.6.0.

Either *It is recommended...* or *We recommend...*

File: README.md [hunk 3]
```diff
@@ -1,2 +1,55 @@
...
+# Step 2-[A]: Build Xilinx Kernel 4.6.0
+Note: You need to follow either Step 2-[A] or 2-[B].
+If you are hope to run Kernel 4.6.0, then run this step. If you are hoping to run Kernel4.9.0, then follow step 2-[B]. However, as we tested, Kernel4.6.0 work properly on QEMU.
+To build Kernel4.6.0, run following commands on terminal opened in axiom-beta-qemu repo folder.
+
+`chmod +x build_kernel4.6.0.sh`
+
+`./build_kernel4.6.0.sh`
+ 
+Note: This will download Xilinx linux kernel newest verion. Therefore you need to have a good internet connection while running this script.
```
Comment (sebix) [inline README.md:25]: > linux kernel newest verion.

*at the* newest ver*s*ion

File: README.md [hunk 4]
```diff
@@ -1,2 +1,55 @@
...
+`chmod +x build_qemu.sh`
+
+`./build_qemu.sh`
+
+Note: This will download QEMU form Xilinx QEMU repo. Therefore you need to have internet conneciton while running this script. Also provide system password to install QEMU.
+
+# Step 2-[A]: Build Xilinx Kernel 4.6.0
+Note: You need to follow either Step 2-[A] or 2-[B].
+If you are hope to run Kernel 4.6.0, then run this step. If you are hoping to run Kernel4.9.0, then follow step 2-[B]. However, as we tested, Kernel4.6.0 work properly on QEMU.
+To build Kernel4.6.0, run following commands on terminal opened in axiom-beta-qemu repo folder.
```
Comment (sebix) [inline README.md:19]: > Kernel4.6.0

missing space
> run following

run *the* following
> on terminal

on *a* terminal

File: README.md [hunk 5]
```diff
@@ -1,2 +1,55 @@
...
+Note: You need to follow either Step 2-[A] or 2-[B].
+This step will build Xilinx Kernel 4.9.0. How ever at the moment, this kernel verion will not run on QEMU properly. Recomend to use Kernel 4.6.0.
+
+To build Kernel 4.9.0, run following commands on terminal opend in axiom-beta-qemu repo folder.
+
+`chmod +x build_kernel4.9.0.sh`
+
+`./build_kernel4.9.0.sh`
+
+Note: This will download Xilinx linux kernel newest verion. Therefore you need to have a good internet connection while running this script.
```
Comment (sebix) [inline README.md:37]: see above

File: README.md [hunk 6]
```diff
@@ -1,2 +1,55 @@
...
+
+# Step 3: Download AXIOM Beta image
+To download AXIOM Beta image, run the following commands.
+
+`chmod +x download_axiom_beta_image.sh`
+
+`./download_axiom_beta_image.sh`
+
+Note: This will download arround 3.7GB. Run following commands on terminal opend in axiom-beta-qemu repo folder
```
Comment (sebix) [inline README.md:46]: missing *the* and *on*, see above

File: README.md [hunk 7]
```diff
@@ -1,2 +1,55 @@
...
+
+# Step 4: Run QEMU
+To run the QEMU use the following commands on terminal opend in axiom-beta-qemu repo folder.
```
Comment (sebix) [inline README.md:49]: see above
s/opend/opened/

File: build_kernel4.6.0.sh [hunk 8]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+echo "	****Cheking for requirements****"
+# requirements Ubuntu
+sudo apt-get install build-essential gcc-arm-linux-gnueabi libglib2.0-dev zlib1g-dev
+
+echo "	****Getting the source****"
+# get the git repo
+https://github.com/Xilinx/linux-xlnx.git
```
Comment (sebix) [inline build_kernel4.6.0.sh:9]: Missing `git clone`

File: README.md [hunk 9]
```diff
@@ -1,2 +1,55 @@
...
+
+`./build_kernel4.6.0.sh`
+ 
+Note: This will download Xilinx linux kernel newest verion. Therefore you need to have a good internet connection while running this script.
+
+# Step 2-[B]: Build Xilinx Kernel 4.9.0
+Note: You need to follow either Step 2-[A] or 2-[B].
+This step will build Xilinx Kernel 4.9.0. How ever at the moment, this kernel verion will not run on QEMU properly. Recomend to use Kernel 4.6.0.
+
+To build Kernel 4.9.0, run following commands on terminal opend in axiom-beta-qemu repo folder.
```
Comment (sebix) [inline README.md:31]: s/opend/opened/

File: README.md [hunk 10]
```diff
@@ -1,2 +1,55 @@
...
+
+`chmod +x build_qemu.sh`
+
+`./build_qemu.sh`
+
+Note: This will download QEMU form Xilinx QEMU repo. Therefore you need to have internet conneciton while running this script. Also provide system password to install QEMU.
+
+# Step 2-[A]: Build Xilinx Kernel 4.6.0
+Note: You need to follow either Step 2-[A] or 2-[B].
+If you are hope to run Kernel 4.6.0, then run this step. If you are hoping to run Kernel4.9.0, then follow step 2-[B]. However, as we tested, Kernel4.6.0 work properly on QEMU.
```
Comment (sebix) [inline README.md:18]: > If you are hope to
Do you mean *If you hope you are able to...*?

> Kernel4.9.0
missing space

> Kernel4.6.0 work properly on QEMU
s/work/works/

**Review by sebix — COMMENTED** [review]
It's already merged, but nevertheless there are a bunch of typos and even a missing command.

File: README.md (hunk 6 above)
Comment (sebix) [reply README.md:46]: s/opend/opened/

File: README.md (hunk 3 above)
Comment (sebix) [reply README.md:25]: s/conneciton/connection/

**Comment by se6astian-pi** [comment]
Thanks sebix, very observant!

File: README.md (hunk 10 above)
Comment (pgsamila) [reply README.md:18]: > If you are hope to
> Do you mean If you hope you are able to...?

I mean, you can use either kernel 4.6.0 or 4.9.0. Step 2-[A] is to kernel 4.6.0 and 2-[B] is to kernel 4.9.0

will update the other mistakes shortly.

**Comment by pgsamila** [comment]
[https://github.com/apertus-open-source-cinema/axiom-beta-qemu/pull/3](url) pull request has updated the typos and missing command.

File: build_kernel4.6.0.sh (hunk 8 above)
Comment (pgsamila) [reply build_kernel4.6.0.sh:9]: updated in [https://github.com/apertus-open-source-cinema/axiom-beta-qemu/pull/3](url)

**Pull request merged**